modules/ui/src/main/scala/lucuma/ui/syntax/time.scala

```diff
@@ -12,12 +12,22 @@ trait time:
      * Format a timespan in the format `${hh}hrs ${mm}mins`
      */
     def toHoursMinutes: String =
-      val hours   = timespan.toHoursPart
+      toHoursMinutes(shortUnits = false)
+
+    /**
+     * Format a timespan in the format `${hh}hrs ${mm}mins` or `${hh}h ${mm}m`
+     */
+    def toHoursMinutes(shortUnits: Boolean = false): String =
```

Comment:
Maybe we should have `toShortHourMinutes`? I imagine people looking often for the meaning of the boolean param?